packages/api-kit/src/utils/config.ts

```diff
@@ -10,6 +10,7 @@ export const TRANSACTION_SERVICE_URLS: Record<string, string> = {
   '42220': 'https://safe-transaction-celo.safe.global',
   '43114': 'https://safe-transaction-avalanche.safe.global',
   '84531': 'https://safe-transaction-base-testnet.safe.global',
+  '84532': 'https://safe-transaction-base-sepolia.safe.global',
```

Comment:
Let's also remove the unsupported one:

```suggestion
  '84532': 'https://safe-transaction-base-sepolia.safe.global',
```